webpush/management/commands/webpush_generate_vapid_keypair.py

```diff
@@ -0,0 +1,10 @@
+from django.core.management.base import BaseCommand
+
+from ...vapid import get_vapid_keypair
+
+
+class GenerateValidKeys(BaseCommand):
```

Comment:
Hi @David-Guillot ,
Thanks for blog article you created about webpush in web project.

Since that release is not yet available, I copied it to my project and tried to run and get ` has no attribute 'Command'`. Class name should be `Command` and not `GenerateValidKeys`

Reply:
Oh :facepalm: sorry about that, I was working on two different codebases when I contributed this PR, and I copied/pasted the version that had this bug. I'll open a PR right away, than you for the report! Next time, don't hesitate to open a PR by yourself :wink: